### DISCUSSION TITLE
对照表排版问题

### DISCUSSION THREAD

**Discussion by lei1320719647** [note286/xduts#78] [Q&A] [answered]
使用内置的基于 longtblr 环境（ tabularray 宏包）实现的符号对照表和缩略语对照表样式该怎么用啊，为啥我修改了style/customize-los = ⟨true|false⟩后，按照手册给的样例排版编译会出错呢

**Comment by lei1320719647** [comment]
谢谢，能否给个对照表文件的模板呢，不知道对照表中内容该怎么写？

**Reply by note286** [reply]
PDF手册中给了啊。

**Reply by note286** [reply]
![image](https://user-images.githubusercontent.com/16207292/226177157-bd1b69e4-667f-4899-ab3c-55e0c76131f5.png)

**Comment by lei1320719647** [comment]
![image](https://user-images.githubusercontent.com/55944911/226177429-fe48b5c4-d752-440b-b774-3a8006df8819.png)

您好，我按照手册中给的模板写的对照表文件，编译的时候出现了下图中的错误，该怎么解决呢？
![image](https://user-images.githubusercontent.com/55944911/226177421-a33f087c-d290-4afc-a14b-77a004a1f641.png)

**Reply by note286** [reply]
README上来就让你更新了，你不更。

**Reply by lei1320719647** [reply]
尴尬了，谢谢，马上去更新，谢谢大佬！

**Comment by note286** [comment, marked as answer]
根据README提示更新LaTeX发行版。

**Comment by lei1320719647** [comment]
上面问题解决啦，谢谢您。还有个问题想请教一下，如何设置表格的宽度跟页面保持一致呢，现在绘制的表格与内容宽度一致，有效甚至还会超出页面。

**Reply by note286** [reply]
新问题单开一个。
